.gitignore

```diff
@@ -12,5 +12,13 @@ public/images/avatars/*
 /config/admin.yml
 /config/config.yml
 /config/database.yml
+/config/tweets-client.ini
+/config/secrets.yml
+/config/skylight.yml
+/*~
+
 .*.swp
 .DS_Store
+*~
+\#*\#
+.\#.*
```

Comment:
@chegejames what exactly are we ignoring here?

Also are we ignoring all the avatars downloaded on adding of politicians?